test-resources/package.json

```diff
@@ -16,6 +16,7 @@
     "dependencies": {
         "@aws-lambda-powertools/commons": "^2.8.0",
         "@aws-lambda-powertools/logger": "^2.8.0",
+        "@aws-sdk/client-dynamodb": "^3.637.0",
```

Comment:
Please can we pin to an exact version? Can't find the doc now but I know we have it somewhere that we should be using set versions